src/components/layout/index.tsx

```diff
@@ -12,6 +12,7 @@ const Wrapper = styled.div`
   main {
     flex: 1;
     padding: 4rem 3rem 10rem 3rem;
+    margin: auto;
```

Comment:
what does this do in the main content?

Reply:
Based on https://css-tricks.com/snippets/css/a-guide-to-flexbox/:

_Example section, first example_
> This relies on the fact a margin set to `auto` in a flex container absorb extra space. So setting a vertical margin of auto will make the item perfectly centered in both axes.

Trade page for instance is exactly the same:

![Trade](https://user-images.githubusercontent.com/43217/64531469-e1eebb00-d30f-11e9-8a91-68071bd8d70d.png)

Although Source code page is now centered...

![SourceCode](https://user-images.githubusercontent.com/43217/64531467-e1562480-d30f-11e9-8ffe-a23f3c37f9e0.png)

Any suggestions on how to center the Widget component?

Reply:
Aha, ok. I understand.

I think for the widget we can just use a negative top margin and centered inside `main`. 

I mean, I would do `main` ocupate all the with, and then the widget can just define it's size, be centered, and use a negative `margin-top`